### ISSUE TITLE
babelHelpers is not defined

### ISSUE THREAD

**Issue by fujinxiang** [algolia/react-element-to-jsx-string#294]
Uncaught ReferenceError: babelHelpers is not defined

when there is something like `style={{margin:10}}`

**Comment by pascalduez** [comment]
Dupe of #11.

Will try to fix in #290

**Comment by pioneer32** [comment]
@pascalduez it doesn't look like #11
I've unpredictably been facing `Uncaught ReferenceError: babelHelpers is not defined` with different components... sometimes removing `defaultProps` helps, sometimes it doesn't...

Here's the line where it fails: 
```
if ((typeof propValue === 'undefined' ? 'undefined' : babelHelpers.typeof(propValue)) === 'symbol') {
```

Unfortunately, I can hardly provide any more details. Will keep posted if I get any information...

**Comment by ArtieReus** [comment]
+1
getting same error:

    // > "Symbols (new in ECMAScript 2015, not yet supported in Flow)"
    // @see: https://flow.org/en/docs/types/primitives/
    // $FlowFixMe: Flow does not support Symbol
    if ((typeof propValue === 'undefined' ? 'undefined' : babelHelpers.typeof(propValue)) === 'symbol') 
    {
    ...

**Comment by VladimirPittner** [comment]
Getting same error:
```
ReferenceError: babelHelpers is not defined
```
with simple:
```js
reactElementToJSXString(<p style={{ color: 'red' }}>Test</p>)
```

Could you please validate that this simple code works for you? So we know if its just specific case for some people.

Thanks

**Comment by pascalduez** [comment]
If any of you want to test the fix from #290 
```
npm i git@github.com:pascalduez/react-element-to-jsx-string.git#hotfix/build
```

**Comment by vvo** [comment]
Just released 14.0.1 with the fix inside it
